src/app/modules/search-input/services/query-parser/query.grammar

```diff
@@ -4,14 +4,15 @@
 // yarn run compile-grammar
 
 @tokens {
-  space { @whitespace+ }
+  space { " " | "\\t" | "\\n" | "\\r" }
```

Comment:
What's the reason for this change?

Reply:
@undsoft  the reason is that using `@whitespace+` will result in compilation errors:
```
Overlapping tokens UnquotedString and space used in same context (example: "") 
```

We have a broad list of available symbols in here, that leads to lezer conflicts with `@whitespace`:
UnquotedString { @asciiLetter+ | $[a-zA-Z\u0080-\uFFFF]+ }
Extracting white spaces from this regex is not possible due to lezer's limitations, at least I tried different ways to do it and no success.

So I decided to override it and it works now.